spec/factories/sequence_factory.rb

```diff
@@ -9,6 +9,16 @@
   sequence(:lon) { Faker::Address.longitude }
   sequence(:name) { Faker::Name.name }
   sequence(:password) { Faker::Lorem.characters(number: 10) }
-  sequence(:french_phone_number) { Faker::PhoneNumber.phone_number }
+  sequence(:french_phone_number) do
+    # Faker generates cell phones that are not valid
+    # because not yet attributed
+    # (Example : 0729999999 and lower are not attributed, 0730000000 and higher are)
+    # We're looping until Faker generates a valid number
+    phone_number = Faker::PhoneNumber.cell_phone
```

Comment:
If faker often gives bad phone numbers, should we use a static phone number?

Reply:
I considered that, but ultimately I don't have a good feeling about generating the same phone number for every user in the test database. Sure, we're not validating the uniqueness of the phone number per user (yet), but I've determined that Faker generates an invalid _cellphone_ number only 1 time out of three, which makes this loop not so much of a performance problem.